vue-components/src/styles/mixins/Label.scss

```
@@ -1,8 +1,12 @@
-@mixin Label {
+@mixin Label( $displayType: 'inline' ) {
     color: $wikit-Label-font-color;
-    padding-block-end: $wikit-Label-padding-block-end;
     font-family: $wikit-Label-font-family;
     font-size: $wikit-Label-font-size;
     font-weight: $wikit-Label-font-weight;
     line-height: $wikit-Label-line-height;
-  }
\ No newline at end of file
+     display: $displayType;
+     
+    @if $displayType == block {
```

Argh, sorry I fudged up the indentation here....

```suggestion
    display: $displayType;
     
    @if $displayType == block {
```

---

It is strange that our style linting is not working here. Also, aren't we supposed to use tabs?^^